CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 
 #### 4.x Releases
 
+## [4.0.5](https://github.com/checkout/frames-ios/releases/tag/4.0.5)
+
+Released on 2023-01-05
+
+Updates:
+
+- Add updates
```

Comment:
Updates:

- Fixed a minor bug with the pay button not becoming disabled once an expiry date is invalid
- Improved UI test cases for the expiry date field to be more extensible
- Updated phone number validation logic, to append the country code to the user's selected country